tabbycat/api/views.py

```diff
@@ -61,3 +65,21 @@ class SpeakerEligibilityView(TournamentAPIMixin, AdministratorAPIMixin, Retrieve
 
     def get_queryset(self):
         return super().get_queryset().prefetch_related('speaker_set')
+
+
+class InstitutionViewSet(TournamentAPIMixin, AdministratorAPIMixin, ModelViewSet):
+    serializer_class = serializers.InstitutionSerializer
+    lookup_field = 'id'
```

Comment:
No need to specify the lookup field as the primary key is the default.

Reply:
This results in an error as the application won't know which argument to look for in the url. 
In urls,it is declared as `<int:id>`
In built-in GenericAPIView, we have 
`lookup_url_kwarg = self.lookup_url_kwarg or self.lookup_field`
So either we change the url declaration to `<int:pk>` or we change the line in question from `lookup_field = 'id'` to `lookup_url_kwarg = 'id`, or keep it as it is. Deleting it is not a viable option. I don't see any reasons for us to prefer one choice over the rest two?

Reply:
Using `<int:pk>` in urls is preferable. Then, we could delete this line ;)